bagitobjecttransfer/recordtransfer/context_processors.py

```diff
@@ -1,15 +1,25 @@
-from recordtransfer.settings import SIGN_UP_ENABLED, MAX_SINGLE_UPLOAD_SIZE, \
-    MAX_TOTAL_UPLOAD_COUNT, MAX_TOTAL_UPLOAD_SIZE, FILE_UPLOAD_ENABLED
+from bagitobjecttransfer.settings.base import (
+    FILE_UPLOAD_ENABLED,
+    MAX_SINGLE_UPLOAD_SIZE,
+    MAX_TOTAL_UPLOAD_SIZE,
+    SIGN_UP_ENABLED,
+)
+from bagitobjecttransfer.settings.base import (
+    MAX_TOTAL_UPLOAD_COUNT,
+)
```

Comment:
Importing the settings like this makes it easier to test:

```suggestion
from django.conf import settings
```

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/265/commits/c510965457dec23d9fcfbb7b45ea77955c52f5ba